lib/box-node-sdk.js

```diff
@@ -176,6 +177,16 @@ BoxSDKNode.prototype.getAppAuthClient = function(type, id) {
 };
 
 /**
+ * Create the in-memory token store instance for the user
+ *
+ * @param {string} userId the ID of the user whose token info can be persisted in-memory.
+ * @returns {TokenStore} A new TokenStore instance
+ */
+BoxSDKNode.prototype.getTokenStore = function(userId) {
+	return new TokenStore(userId);
+};
```

Comment:
I would simply provide a reference to the class here, so the client can do whatever they want with it (not everyone will want to use userID necessarily.

```js
BoxSDKNode.MemoryTokenStore = MemoryTokenStore;
```